src/components/Button/Button.jsx

```diff
@@ -1,26 +1,26 @@
-import Button from '@mui/material/Button';
+import css from './Button.module.css';
 
-const MainButton = ({ type, content, onClick = null, disabled = false, hasaccent = false }) => {
-  const setBgColor = hasaccent => (hasaccent ? `#24CCA7` : `#ffffff`);
-  const setColor = hasaccent => (hasaccent ? `#ffffff` : `#4A56E2`);
-  const setBorder = hasaccent => (hasaccent ? `#24CCA7` : `#4A56E2`);
+const Button = ({ type, content, onClick = null, idDisabled = false, hasAccent = false }) => {
```

Comment:
nie o to chodziło, żebyś nazwała przycisk Button, tylko żebyś nazwała go inaczej. 
Ponieważ, jak w projekcie używamy tez MUI <Button />, to będzie konflikt nazw - rozumiesz? Dlatego Twój przycisk trzeba nazwać inaczej.